src/connectors/index.ts

```diff
@@ -110,11 +111,15 @@ const zenGoBN = zenGoModule({
     },
 })
 
-// const walletLink = walletLinkModule({ darkMode: true })
-
 export const onboard = init({
     wallets: [injectedBN, walletConnectBN, zenGoBN, coinbaseWalletSdk],
     chains: [
+        {
+            id: '0xa4ec',
+            token: 'CELO',
+            label: 'Celo',
```

Comment:
Looks like first chain in list. On both mobile metamask / gd wallet still 122 is being requested to connect to